samples/event_manager/prj.conf

```diff
@@ -0,0 +1,30 @@
+# Enabling assert
+CONFIG_ASSERT=y
+
+# Use RTT
+CONFIG_UART_CONSOLE=n
+CONFIG_USE_SEGGER_RTT=y
+CONFIG_RTT_CONSOLE=y
+
+# Configuration required by Event Manager
+CONFIG_EVENT_MANAGER=y
+CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
+CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=1024
+CONFIG_HEAP_MEM_POOL_SIZE=2048
+CONFIG_REBOOT=y
+
+# Logger configuration
+CONFIG_LOG=y
+CONFIG_LOG_DEFAULT_LEVEL=2
+CONFIG_LOG_BACKEND_RTT=y
+CONFIG_LOG_BACKEND_RTT_MODE_DROP=y
+CONFIG_LOG_MODE_NO_OVERFLOW=y
+CONFIG_LOG_PRINTK=y
+CONFIG_LOG_PRINTK_MAX_STRING_LENGTH=128
```

Comment:
In a sample, I would expect to set only the required (bare minimum) configuration values and rely mostly on defaults.

Reply:
changed